tests/test_user_document.py

```diff
@@ -62,35 +58,36 @@ def test_post_patient_report(self):
         create_service_token(client=client, user=get_user(TEST_USER_ID))
         self.login()
 
-        test_contents = "This is a test."
+        test_contents = 'This is a test.'
         with NamedTemporaryFile(
             prefix='udoc_test_',
             suffix='.pdf',
             delete=True,
         ) as temp_pdf:
-            temp_pdf.write(test_contents)
+            temp_pdf.write(test_contents.encode('utf-8'))
             temp_pdf.seek(0)
             tempfileIO = BytesIO(temp_pdf.read())
-            rv = self.client.post('/api/user/{}/patient_report'.format(TEST_USER_ID),
+            response = self.client.post('/api/user/{}/patient_report'.format(
+                TEST_USER_ID),
                                 content_type='multipart/form-data', 
                                 data=dict({'file': (tempfileIO, temp_pdf.name)}))
-            self.assert200(rv)
+            assert response.status_code == 200
         udoc = db.session.query(UserDocument).order_by(UserDocument.id.desc()).first()
         fpath = os.path.join(current_app.root_path,
                             current_app.config.get("FILE_UPLOAD_DIR"),
                             str(udoc.uuid))
         with open(fpath, 'r') as udoc_file:
-            self.assertEqual(udoc_file.read(), test_contents)
+            assert udoc_file.read() == test_contents
         os.remove(fpath)
 
-        self.assertEqual(udoc.user_id, TEST_USER_ID)
-        self.assertEqual(udoc.intervention.description,
-                          INTERVENTION.SEXUAL_RECOVERY.description)
+        assert udoc.user_id == TEST_USER_ID
+        assert (udoc.intervention.description
+                == INTERVENTION.SEXUAL_RECOVERY.description)
 
 
     def test_download_user_document(self):
         self.login()
-        test_contents = "This is a test."
+        test_contents = b'This is a test.'
```

Comment:
Please change back to native string :) (you'll likely need to change `BytesIO` as well)

Reply:
Probably going to run into the same issue as above, please repeat above solution here